.gitattributes

```diff
@@ -30,3 +30,6 @@ src/backend/distributed/utils/ruleutils_10.c -citus-style
 src/backend/distributed/utils/ruleutils_11.c -citus-style
 src/backend/distributed/utils/ruleutils_12.c -citus-style
 src/include/distributed/citus_nodes.h -citus-style
+
+# Hide diff on github by default for copied udfs
+src/backend/distributed/udfs/*/[123456789]*.sql linguist-generated=true
```

Comment:
This doesn't seem to have the desired effect.

The format is `x.y-z--x.y-z.sql` I guess, you are only matching numbers it seems.

Reply:
The * is a blob, so this matches any file starting with a digit

Reply:
I mean `x.y-z.sql` anyway, I see the diff in github :)

Reply:
it's not regex but git matching, so it matches all sql files where the first character is a number. I didn't update the path to `.../sql/udfs/...` though so that's why the matching is not working